README.md

```diff
@@ -0,0 +1,58 @@
+Codelicia\Immutable
+===================
+
+Install the `ImmutableProperties` library and get to work with immutable properties.
+
+### Installation
+
+```
+$ composer req codelicia/immutability --sort-packages
```

Comment:
`require`. Also remove `--sort-packages`: that's up for the user to decide, plus the flag may change over time, and it's not up to this package to document composer usage ;-)